addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -73,8 +154,160 @@ def _stripe_make_request(self, endpoint, payload=None, method='POST', offline=Fa
+            'company[address][postal_code]': self.company_id.zip or '',
+            'company[address][state]': self.company_id.state_id.name or '',
+            'company[name]': self.company_id.name,
+            'individual[address][city]': self.company_id.city or '',
+            'individual[address][country]': self.company_id.country_id.code or '',
+            'individual[address][line1]': self.company_id.street or '',
+            'individual[address][line2]': self.company_id.street2 or '',
+            'individual[address][postal_code]': self.company_id.zip or '',
+            'individual[address][state]': self.company_id.state_id.name or '',
+            'individual[email]': self.company_id.email or '',
+            'business_profile[name]': self.company_id.name,
+        }
+
+    def _stripe_account_link_url(self, connected_account_id, menu_id):
+        """ Create an account link and return its URL.
+
+        An account link url is the beginning URL of Stripe Onboarding.
+        This URL is only valid once, and can only be used once.
+
+        :param str connected_account_id: The id of the connected account
+        :param int menu_id: The id of the menu to return to after onboarding.
+        :return: The account link URL
+        :rtype: str
+        """
+        return_url = urls.url_join(
+            self.company_id.get_base_url(),
+            StripeController._onboarding_return_url + f'{self.id}/{menu_id}'
+        )
+        refresh_url = urls.url_join(
+            self.company_id.get_base_url(),
+            StripeController._onboarding_refresh_url + f'{self.id}/{connected_account_id}/{menu_id}'
+        )
+        account_link = self._stripe_make_proxy_request('account_links', payload={
+            'account': connected_account_id,
+            'return_url': return_url,
+            'refresh_url': refresh_url,
+            'type': 'account_onboarding',
+        })
+        return account_link['url']
+
+    def _stripe_make_proxy_request(self, endpoint, payload=None, version=1):
+        """ Make a request to the Stripe proxy at the specified endpoint.
+
+        :param str endpoint: The proxy endpoint to be reached by the request
+        :param dict payload: The payload of the request
+        :param int version: The proxy version used
+        :return The JSON-formatted content of the response
+        :rtype: dict
+        :raise: ValidationError if an HTTP error occurs
+        """
+        proxy_payload = {
+            'jsonrpc': '2.0',
+            'id': uuid.uuid4().hex,
+            'method': 'call',
+            'params': {'payload': payload},
+        }
+        url = urls.url_join(PROXY_URL, f'{version}/{endpoint}')
+        try:
+            response = requests.post(url=url, json=proxy_payload, timeout=60)
+        except requests.exceptions.ConnectionError:
+            raise ValidationError(
+                _("Stripe Proxy: Could not establish the connection to the API.")
```

Comment:
The proxy is offline in that case, not Stripe API.
```suggestion
                _("Stripe Proxy: Could not establish the connection.")
```